tests/DataSourcePouchDBTests.js

```diff
@@ -168,10 +170,35 @@ fluid.defaults("kettle.tests.dataSource.15.CouchDB.URL.set.existing", {
     }
 });
 
+fluid.defaults("kettle.tests.dataSource.16.CouchDB.URL.set.specialChars", {
```

Comment:
Hi cindyli - it looks like this test is still in the "pouchDB" namespace and test file rather than being moved to plain "DataSourceURLTests.js" as it should be.